### DISCUSSION TITLE
you forgot the sound folder

### DISCUSSION THREAD

**Discussion by OldGamers** [ikemen-engine/Ikemen-GO#1163] [Feature Request]
### Is your feature request related to a problem? Please describe.

when I downloaded the 99.1 RC1 you guys forgot to add the sound folder 
![Untitled](https://user-images.githubusercontent.com/35203194/231155031-7d12c3c1-8439-4ccf-9925-f3aaee7c078b.png)

**Comment by potsmugen** [comment]
It's probably intentional since that folder came empty anyway.

**Comment by K4thos** [comment]
added to screenpack repo
